### ISSUE TITLE
Very long strings in code blocks run off end of page

### ISSUE THREAD

**Issue by vhsdream** [noraj/OSCP-Exam-Report-Template-Markdown#21]
When using code blocks with very long strings (example image attached), the strings eventually wrap, but do so after the end of the block, spilling over into the white pat of the page. I have looked up this issue in pandoc and the eisvogel template, and it appears that there is a workaround in the .tex file (at line 337) but it does not work completely. Does anyone have a workaround for the workaround?

![code_block](https://user-images.githubusercontent.com/67816022/131576377-641ca83e-f1fe-453b-9850-2d7dafa456a7.png)

**Comment by noraj** [comment]
Yeah and for even longer you could even encounter errors https://github.com/Wandmalfarbe/pandoc-latex-template/issues/122
There is currently no real solution. The workaround is to manually split your long lines.

**Comment by noraj** [comment]
Closing since upstream is a wontfix.
